pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+from ctypes import POINTER
+from ctypes import byref
+from ctypes import c_bool
+from ctypes import c_char_p
+from ctypes import c_int
+from enum import Enum
+
+import pyaedt
+
+
+class FilterType(Enum):
+    """Selects type of filter with the associated mathematical formulation.
+
+    Attributes:
+    GAUSSIAN: Represents a Gaussian filter.
+    BESSEL: Represents a Bessel filter.
+    BUTTERWORTH: Represents a Butterworth filter.
+    LEGENDRE: Represents a Legendre filter.
+    CHEBYSHEV_I: Represents a Chevyshev type I filter.
+    CHEBYSHEV_II: Represents a Chevyshev type II filter.
+    HOURGLASS: Represents a Hourglass filter.
+    ELLIPTIC: Represents a elliptic filter.
+    Custom, rasied-cos, matched, and delay filter types are not available in this release.
```

Comment:
```suggestion
    
    Custom, raised-cos, matched, and delay filter types are not available in this release.
```
Blank line is necessary for doc to render correctly. Should cos be spelled out. I know that I don't know what this means.